docs/Parallel-Learning-Guide.rst

```diff
@@ -489,12 +489,15 @@ Example
 Ray
 ^^^
 
-`Ray`_ is a Python-based framework for distributed computing. The `lightgbm_ray`_ project, maintained within the official Ray GitHub organization, can be used to perform distributed LightGBM training using ``ray``.
+`Ray`_ is a Python-based framework for distributed computing. Ray provides LightGBM support through the Ray Train API with ``LightGBMTrainer``, in addition to the `lightgbm_ray`_ project maintained within the official Ray GitHub organization.
 
-See `the lightgbm_ray documentation`_ for usage examples.
+For the Ray Train API, see `the Ray documentation`_ for usage examples.
+
+For the lightgbm_ray project, see `the lightgbm_ray documentation`_ for usage examples.
 
 .. note::
 
+  ``ray`` is not maintained by LightGBM's maintainers. Bug reports or feature requests should be directed to https://github.com/ray-project/ray/issues.
```

Comment:
Instead of adding another almost-identical sentence, please just change the existing line to being "`lightgbm_ray` and `ray` are not..."